edb/edgeql/compiler/viewgen.py

```diff
@@ -1202,11 +1202,15 @@ def _inline_type_computable(
     stype: s_objtypes.ObjectType,
     compname: str,
     propname: str,
+    is_mutation: bool,
     *,
     shape_ptrs: List[ShapePtr],
     ctx: context.ContextLevel,
 ) -> None:
     assert isinstance(stype, s_objtypes.ObjectType)
+    # Injecting into non-view objects /usually/ works, but it fails if the
```

Comment:
The "usually works" part is wrong too, because user-defined schema types shouldn't be mutated either (if you add or change anything, it should be a new type id), so this is a good change.